### PR TITLE
Feature/issue #7 credential management api

### LINKED ISSUE #7: [Task Assignment]: Credential Management API Implementation
### Task Description
Implement a web project using HTML, CSS, and JavaScript that integrates a Credential Management API.

### Task Requirements

- Create a new directory inside "participants" with your name.
- Develop necessary HTML, CSS, and JavaScript files.
- Integrate a Credential Management API into your project.
- Update bootcamp.html with your name under `<navigation>`

### Submission Guidelines

- Create a branch: git checkout -b feature/your-feature-name
- Commit changes: git commit -m "Description of changes"
- Push to branch: git push origin feature/your-feature-name
- Create a Pull Request to merge changes into main.

### Additional Guidelines

- Maintain code quality and organization.
- Implement error handling and thorough testing.
- Document external APIs used.

### Resources
[Credential Management API Documentation](https://developer.mozilla.org/en-US/docs/Web/API/Credential_Management_API)

### PR DESCRIPTION
**Pull request by rashilakhadka**
### Description
This is a sample project that utilizes Credentials Management API.

### Changes Made
- Added username(rashilakhadka) as a list on the bootcamp.html file which contains the project link
- Added a folder participants folder named rashilakhadka(username) as suggested which contains the project

### Related Issues

[Credential Management API Implementation](https://github.com/org-45/yantra-lite/issues/7)
Resolves #7

### Checklist

- [x] I have tested these changes locally.
- [x] I have added/updated documentation as needed.
- [x] I have reviewed my code for potential issues.
- [x] I have followed the project's coding guidelines.
- [x] I have updated relevant comments and commit messages.

### Screenshots (if applicable)
image
image
image
image

### Reviewer Request
@ankman007 

### By submitting this PR, I confirm that

- [x] I have read and understood the project's contributing guidelines.
- [x] This PR is ready for review and merging.
- [x] I understand that automated tests should pass before merging.

### CODE REVIEW

**Review by manjilkoirala — CHANGES_REQUESTED** [review]
The code seems good. Please link your issue to the PR description.

**Comment by ankman007** [comment]
@rashilakhadka Change "issue 123" to the issue you are solving.

**Comment by rashilakhadka** [comment]
> The code seems good. Please link your issue to the PR description.

Done.

**Comment by rashilakhadka** [comment]
> @rashilakhadka Change "issue 123" to the issue you are solving.

Done

File: participants/rashilakhadka/script.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    const logoutBtn = document.getElementById('logout');
+    const feedback = document.getElementById('feedback');
+
+    //check for credential support in browser
+    if(!navigator.credentials || !navigator.credentials.preventSilentAccess) {
+        feedback.textContent = 'Credential Management API is not supported in this browser';
+        saveCredentialsBtn.style.display = 'none';
+        logoutBtn.style.display = 'none';
+        return;
+    }
```
Comment (pfieffer) [inline participants/rashilakhadka/script.js:13]: The early exit in case if the credential support is not available is a very good practice. 🚀

**Review by pfieffer — APPROVED** [review]
LGTM
Amazing work @rashilakhadka 
But please do sync your branch to get latest changes, resolve any conflicts and push.

**Review by manjilkoirala — APPROVED** [review]

**Comment by manjilkoirala** [comment]
Resolve the conflict then merge.
Thank you for the PR @rashilakhadka ☺️

**Pull request merged**